deploy/example/sharedisk/README.md

```diff
@@ -0,0 +1,77 @@
+# Shared disk example
+
+[Azure shared disks](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-shared) (preview) is a new feature for Azure managed disks that enables attaching an Azure managed disk to multiple virtual machines (VMs) simultaneously. Attaching a managed disk to multiple VMs allows you to either deploy new or migrate existing clustered applications to Azure.
+
+1. Create Storage Class and PVC
+
+```yaml
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: disk.csi.azure.com
+provisioner: disk.csi.azure.com
+parameters:
+  skuname: Premium_LRS  # Currently shared disk only available with premium SSD
+  maxShares: "2"
+  cachingMode: None  # ReadOnly host caching is not available for premium SSD with maxShares>1
+reclaimPolicy: Delete
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvc-azuredisk
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 256Gi  # minimum size of share disk is 256GB (P15)
+  storageClassName: disk.csi.azure.com
+```
+
+2. Create a deployment with 2 replicas and specify `volumeDevices`, `devicePath` in Spec
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nginx
+  name: deployment-azuredisk
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+      name: deployment-azuredisk
+    spec:
+      containers:
+        - name: deployment-azuredisk
+          image: nginx
+          volumeDevices:
+            - name: azuredisk
+              devicePath: /dev/sdx
+          volumeMounts:
+            - name: azuredisk
+              mountPath: "/mnt/azuredisk"
+              readOnly: false
+      volumes:
+        - name: azuredisk
+          persistentVolumeClaim:
+            claimName: pvc-azuredisk
+```
+
+3. Check block device in pod
+
+```console
+# kubectl exec -it deployment-sharedisk-7454978bc6-xh7jp bash
```

Comment:
Since this is shared disk doc, it would be great to have an example of the how the contents are written from replica and then can be ready from other replica. ie. on  replica designated as write replica - dd if=<some real content> and then on read replica dd if=/dev/sdx and see the content.

Reply:
added a note, from driver level, we cannot grantee single write, multi read